src/settings-ui/Settings.UI/SettingsXAML/Views/GeneralPage.xaml

```diff
@@ -150,14 +152,14 @@
                                     x:Uid="General_TryAgainToDownloadAndInstall"
                                     Command="{Binding UpdateNowButtonEventHandler}"
                                     IsEnabled="{Binding IsDownloadAllowed, Mode=OneWay}"
-                                    Visibility="{Binding Mode=OneWay, Path=IsNewVersionDownloading, Converter={StaticResource BoolNegationConverter}}" />
+                                    Visibility="{x:Bind ViewModel.IsNewVersionDownloading, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}" />
 
                                 <!--  In progress panel  -->
                                 <StackPanel
                                     Margin="0,0,0,16"
                                     Orientation="Horizontal"
                                     Spacing="18"
-                                    Visibility="{Binding Mode=OneWay, Path=IsNewVersionDownloading, Converter={StaticResource BoolToVisibilityConverter}}">
+                                    Visibility="{x:Bind ViewModel.IsNewVersionDownloading, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}">
```

Comment:
Just curious; is the additional `ViewModel` needed, would it work without it as well or does the `x:Bind` need it to look it up?

Reply:
I had the exact same idea. It's noted in the pr comment...

Reply:
Tricky question. I tried removing the "Viewmodel" everywhere, but VS was not really about that. Is probably not _needed_ right here, because obviously it (currently) works without it. This got added via Search&Replace + reverting a commit.
If there's a way to _generally_ remove the prefix without any issues, we could try and add that.